pkgs/applications/misc/dmenu-rs/default.nix

```diff
@@ -51,21 +51,11 @@ stdenv.mkDerivation rec {
     lockFile = ./Cargo.lock;
   };
 
-  # The rust-xcb dependency dynamically generates rust code at build time.
-  # This derivation uses nixpkgs rust functions that vendor each cargo
-  # dependency's source code into the READ-ONLY nix store. To avoid the code
-  # generation step failing, we copy the rust-xcb source out of the nix store
-  # and make it writeable. Also, we remove the build's hardcoded c compiler.
-  # See: https://github.com/rust-x-bindings/rust-xcb/tree/v0.8.2
+  # Copy the Cargo.lock stored here in nixpkgs into the build directory.
+  # Also, fix an bug in the makefile (insert a tab indented line at line 56).
+  # See https://github.com/Shizcow/dmenu-rs/pull/50
   postPatch = ''
-    substituteInPlace config.mk --replace "clang" ""
-
-    chmod +w "$NIX_BUILD_TOP/cargo-vendor-dir"
-    mkdir -p "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2-readwrite"
-    cp -r --no-preserve=mod "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2/." "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2-readwrite"
-    unlink "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2"
-    mv "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2-readwrite" "$NIX_BUILD_TOP/cargo-vendor-dir/xcb-0.8.2"
-
+    sed -i "56i\\\tcp src/man/src/stest.1 target/" makefile
```

Comment:
could you use `fetchpatch` and add the patch to `patches` instead?

Reply:
Done in 5765305de256f6751a4299bf87899b67645fec5d